data/json/recipes/recipe_food.json

```diff
@@ -85,6 +85,33 @@
     "tools": [ [ [ "water_purifier", 14 ], [ "pur_tablets", 1 ], [ "char_purifier", 12 ], [ "char_purifier_clay", 12 ] ] ],
     "components": [ [ [ "water", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "water",
+    "category": "CC_FOOD",
+    "id_suffix": "using_water_purifier",
+    "subcategory": "CSC_FOOD_DRINKS",
+    "skill_used": "survival",
+    "time": "1 m",
+    "autolearn": true,
+    "tools": [ [ [ "pur_tablets", 1 ] ] ],
+    "components": [ [ [ "water_murky", 1 ] ] ]
```

Comment:
```suggestion
    "components": [ [ [ "water_murky", 1 ] ], [ [ "pur_tablets", 1 ] ] ]
```

Reply:
Because charge is removed from the tablet, it needs to be listed as component.

Reply:
ah yeah thanks, not sure what i was thinking there.